toolkit/scripts/containerized-build/resources/_local_repo

```diff
@@ -0,0 +1,17 @@
+[host_build_repo]
```

Comment:
Any reason this file is named `_local_repo` and not `local.repo`? You rename it in the .Dockerfile anyway.

Reply:
We already have a local.repo in Mariner toolkit. Renamed _local_repo.not_a_repo to local_repo.disabled_repo